gnome-extra/gnome-boxes/gnome-boxes-43.0.ebuild

```diff
@@ -0,0 +1,121 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+VALA_MIN_API_VERSION="0.40"
+
+inherit gnome.org gnome2-utils linux-info meson readme.gentoo-r1 vala xdg
+
+DESCRIPTION="Simple GNOME application to access remote or virtual systems"
+HOMEPAGE="https://wiki.gnome.org/Apps/Boxes"
+
+LICENSE="LGPL-2+ CC-BY-2.0"
+SLOT="0"
+
+KEYWORDS="~amd64"
+
+# FIXME: qemu probably needs to depend on spice[smartcard] directly with USE=spice
+# FIXME: Check over libvirt USE=libvirtd,qemu and the smartcard/usbredir requirements
+# Technically vala itself still ships a libsoup vapi, but that may change, and it should be better to use the .vapi from the same libsoup version
+DEPEND="
+	>=app-arch/libarchive-3.0.0:=
+	>=dev-libs/glib-2.52:2
+	>=x11-libs/gtk+-3.24.1:3
+	>=gui-libs/libhandy-1.5.0:1=
+	>=sys-libs/libosinfo-1.10.0
```

Comment:
This depends on libsoup:2.4. We need to make revbump that uses libsoup:3.0... and probably update reverse dependencies...

Looks like the only revdeps of libosinfo are

- app-emulation/virt-manager
-- Doesn't depend on libsoup, so nothing to do.
- app-misc/tracker-miners
-- Doesn't depend on libsoup, so nothing to do.
- gnome-extra/gnome-boxes
-- Need to update gnome-boxes-42 to depend on libsoup:2.4-libosinfo.

So not much to do \o/